packages/fetch-http-handler/src/stream-collector.spec.ts

```diff
@@ -29,4 +29,16 @@ describe("streamCollector", () => {
       done();
     });
   });
+
+  it("returns a Uint8Array when blob is empty", done => {
+    const expected = new Uint8Array(0);
+
+    const dataPromise = new Response(expected.buffer)
+      .blob()
+      .then(blob => streamCollector(blob));
+    dataPromise.then((data: any) => {
+      expect(data).toEqual(Uint8Array.from([]));
```

Comment:
```suggestion
      expect(data).toEqual(expected);
```

Reply:
fixed in [commit](https://github.com/aws/aws-sdk-js-v3/pull/1199/commits/1e5fa454a42f0219eca536d2356f7191804cabb0)